### ISSUE TITLE
Error: Cannot read property 'depth' of undefined

### ISSUE THREAD

**Issue by voodooattack** [pnpm/supi#15]
This bug is happening as part of a huge project that uses supi to install packages. I'm not sure of how reproducible this is outside of it, but I can provide access to the sources on request.

### supi version:
0.6.1

### Code to reproduce the issue:

```ts
  const supiOptions = {
    prefix,
    storePath,
    registry,
    production: true
  };
  await installPkgs(
    queue.map(({ pkg }) => `${pkg.name}@${pkg.version}`),
    supiOptions
  );
```

### Expected behavior:
Install the packages.

### Actual behavior:
I get this error:

```
TypeError: Cannot read property 'depth' of undefined
    at resolvePeersOfNode (/home/voodooattack/Desktop/cloudpress.io/@cloudpress/base-engine/node_modules/supi/src/link/resolvePeers.ts:96:70)
    at resolvePeersOfChildren (/home/voodooattack/Desktop/cloudpress.io/@cloudpress/base-engine/node_modules/supi/src/link/resolvePeers.ts:197:31)
    at resolvePeersOfNode (/home/voodooattack/Desktop/cloudpress.io/@cloudpress/base-engine/node_modules/supi/src/link/resolvePeers.ts:102:42)
    at resolvePeersOfChildren (/home/voodooattack/Desktop/cloudpress.io/@cloudpress/base-engine/node_modules/supi/src/link/resolvePeers.ts:197:31)
    at Object.default_1 [as default] (/home/voodooattack/Desktop/cloudpress.io/@cloudpress/base-engine/node_modules/supi/src/link/resolvePeers.ts:68:3)
    at Object.<anonymous> (/home/voodooattack/Desktop/cloudpress.io/@cloudpress/base-engine/node_modules/supi/src/link/index.ts:61:40)
    at Generator.next (<anonymous>)
    at /home/voodooattack/Desktop/cloudpress.io/@cloudpress/base-engine/node_modules/supi/lib/link/index.js:7:71
    at new Promise (<anonymous>)
    at __awaiter (/home/voodooattack/Desktop/cloudpress.io/@cloudpress/base-engine/node_modules/supi/lib/link/index.js:3:12)
    at Object.default_1 [as default] (/home/voodooattack/Desktop/cloudpress.io/@cloudpress/base-engine/node_modules/supi/lib/link/index.js:29:12)
    at /home/voodooattack/Desktop/cloudpress.io/@cloudpress/base-engine/node_modules/supi/src/api/install.ts:498:36
    at Generator.next (<anonymous>)
    at fulfilled (/home/voodooattack/Desktop/cloudpress.io/@cloudpress/base-engine/node_modules/supi/lib/api/install.js:4:58)
    at <anonymous>
```

### Additional information:

 - `node -v` prints: v8.9.0
 - Windows, OS X, or Linux?: Ubuntu 17.04

**Comment by zkochan** [comment]
Reproduced. You can expect a fix for this.

**Comment by zkochan** [comment]
Fix published with `supi@0.7.1`.

not published to pnpm yet.

by the way, supi had a breaking change. Now it requires `@pnpm/logger` to be installed instead of `pnpm-logger`.

**Comment by voodooattack** [comment]
@zkochan That was fast! Thank you!
